core/src/main/java/org/elasticsearch/common/cli/Terminal.java

```diff
@@ -157,13 +157,13 @@ public void printStackTrace(Throwable t) {
         private final PrintWriter printWriter = new PrintWriter(System.out);
 
         @Override
-        public void doPrint(String msg, Object... args) {
-            System.out.print(String.format(Locale.ROOT, msg, args));
+        public void doPrint(String msg) {
+            System.out.print(String.format(Locale.ROOT, "%s", msg));
         }
```

Comment:
This can just be `System.out.print(msg)` now I think. Thanks for removing the formatting from these print apis! I think that was the real problem, its trappy for a `printf()` to be named anything other than `*printf()`. And in this case the caller can just always `String.format` themselves.